notes/README.md

```diff
@@ -0,0 +1,592 @@
+
+| Property | Required | Notes |
+| -- | -- | --
+| incomingAmount | No | If specified, the assetCode and assetScale MUST match those of the underlying Account.
+| expiresAt | No | MUST be a time in the future. The resource server MAY override the provided value to be within accepted bounds.
+| description | No |
+| externalRef | No |
+| receiptNonce | No | If provided, with a receiptSecret, the receiver SHOULD enable STREAM receipts.
+| receiptSecret | No | If provided, with a receiptNonce, the receiver SHOULD enable STREAM receipts.
+
+If any other properties are provided the request MUST be rejected.
+
+#### 2.2.2 Get Incoming Payment
+
+**Method**: GET
+**Accept**: application/op-incoming-payment-v1+json
+
+Returns the current state of the incoming payment.
+
+#### 2.2.3 Complete Incoming Payment
+
+**Method**: PUT
+**Content**-Type: application/op-incoming-payment-v1+json
+**Accept**: application/op-incoming-payment-v1+json
+
+If the state is **Pending** or **Processing** then the state can be updated to **Completed** by a client that has the appropriate permissions.
+
+No further payments will be accepted by the ASPSP after the payment has been transitioned to the **Completed** state.
+
+The request body contains an incoming payment resource encoded as JSON with only the following properties: 
+
+| Property | Required | Notes |
+| -- | -- | --
+| state | Yes | Only a value of completed is permitted and only if the client has permission to complete incoming payments.
+
+If any other properties are provided the request MUST be rejected.
+
+### 3. Outgoing Payment
+
+An outgoing payment is a sub-resource of an account. It represents a payment from the account and will transition through various states until it is either completed, rejected or failed.
+
+A payment has a number of states indicating the status of the payment.
+
+1. Pending
+2. Rejected
+3. Authorized
+4. Completed
+5. Expired
+6. Failed
+
+By default a new payment will have the state of **Pending**. A client with permission to authorize the payment can update the state to **Authorized** following which the ASPSP will attempt to execute the payment and the client can no longer update the payment.
+
+When creating a new payment the client must provide at least one of, a value for the **sendAmount**, a value for the **receiveAmount**, or an incoming payment URL as the value for the **receiver**.
+
+If the client provides an incoming payment URL the ASPSP must query it to determine the **receiveAmount**. If the ASPSP is unable to get a **receiveAmount** from the URL provided (insufficient permissions, or the URL is not for a valid incoming payment in an appropriate state) then the ASPSP should set the state of the Outgoing Payment to **Failed**.
+
+If the client provides a **receiveAmount** or incoming iayment then the ASPSP SHOULD calculate a **sendAmount** that will be debited from the underlying account and set this in the response.
```

Comment:
```suggestion
If the client provides a **receiveAmount** or incoming payment then the ASPSP SHOULD calculate a **sendAmount** that will be debited from the underlying account and set this in the response.
```

Reply:
[Rafiki](https://github.com/interledger/rafiki/blob/main/docs/transaction-api.md#outgoing-payment-lifecycle) currently calculates the outgoing payment quote asynchronously after the payment is created because the quoting process isn't guaranteed to complete before the create outgoing payment request times out.
Maybe this could also have an optional `Quoting` state before the payment is `Pending`?